pkg/controller/kogitosupportingservice/kogitodataindex_reconciler.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kogitosupportingservice
+
+import (
+	appv1alpha1 "github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure/services"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	controller1 "sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"time"
+)
+
+// DataIndexSupportingServiceResource implementation of SupportingServiceResource
+type DataIndexSupportingServiceResource struct {
+}
+
+// Reconcile reconcile Data Index
+func (*DataIndexSupportingServiceResource) Reconcile(client *client.Client, instance *appv1alpha1.KogitoSupportingService, scheme *runtime.Scheme) (reconcileAfter time.Duration, err error) {
+	log.Infof("Reconciling KogitoDataIndex for %s in %s", instance.Name, instance.Namespace)
```

Comment:
Seems to be a duplicit log entry, in the log I see:
```
{"level":"info","T":"2020-11-05T17:20:54.598Z","logger":"kogitosupportingservice_controller","msg":"Reconciling KogitoSupportingService for data-index in ksuta-test"}
{"level":"info","T":"2020-11-05T17:20:54.608Z","logger":"kogitosupportingservice_controller","msg":"Reconciling KogitoDataIndex for data-index in ksuta-test"}
```
I think this log can be removed.